src/styles/blocks/button-wrapper.scss

```diff
@@ -0,0 +1,41 @@
+@import "../utils/variables";
```

Comment:
Generally it would be a good idea to group up blocks into folders based on main elements like header and main, for example:
![image](https://github.com/mate-academy/layout_miami/assets/87986955/aabc78b1-36a4-4b13-a1b5-e0be33372d66)

Reply:
I grouped blocks.